examples/cloudtrail/main.tf

```diff
@@ -31,7 +31,7 @@ module "observe_lambda" {
 }
 
 module "observe_lambda_s3_subscription" {
-  source = "../../s3_bucket_subscription"
+  source = "../..//modules/s3_bucket_subscription"
```

Comment:
seems like a duplicated '/'

Reply:
The `//` simply denotes the boundary between repo and submodule. It shouldn't behave differently, other than avoiding copying over files if the root repo has already been cloned.